.github/workflows/npm-push.yml

```diff
@@ -0,0 +1,44 @@
+name: NodeJS CI with NPM (Merge)
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  check-pr-labels:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: read
+    outputs:
+      labels: ${{ steps.get-pr.outputs.pr_labels }}
+    steps:
+    - name: Get Pull Request information
+      uses: 8BitJonny/gh-get-current-pr@2.2.0
+      id: get-pr
+
+  build:
+    runs-on: ubuntu-latest
+    needs: [check-pr-labels]
+    if: contains(needs.check-pr-labels.outputs.labels, 'release')
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - run: npm ci
+      #- run: npm test
+    
+  publish-npm:
+    runs-on: ubuntu-latest
+    needs: [build]
+    if: contains(needs.check-pr-labels.outputs.labels, 'release')
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          registry-url: https://registry.npmjs.org/
+      - run: npm ci
+      - run: npm publish
```

Comment:
Since this takes care of publishing, is the [npm-publish workflow](https://github.com/sngular/openapi-nodejs-cli/blob/main/.github/workflows/npm-publish.yml) needed anymore? That one runs when a release is manually created

Reply:
@jorgecasar will decide this. I'm only adding the feature of checking the labels of the PRs, but the previous functionality of releasing manually and publishing the release with the action has not been edited by my side.

Reply:
Ok, so I think the better idea is to let this as it is, and when you can publish on the Sngular namespace, you can retake this.

Reply:
@jorgecasar if you agree, we could let this PR open until the namespace work.

Reply:
We can merge it